runtime/vm/service/service.md

```diff
@@ -615,8 +615,8 @@ reference will be returned.
 
 ```
 AllocationProfile getAllocationProfile(string isolateId,
-                                       boolean reset [optional],
-                                       boolean gc [optional])
+                                       bool reset [optional],
```

Comment:
Thanks! Can you make the same changes in `service_dev.md` as well?

Reply:
done 👍